src/main/java/Chat/tasks/Evelyn.java

```diff
@@ -0,0 +1,261 @@
+package Chat.tasks;
+
+import Chat.exceptions.RepeatMark;
+import Chat.exceptions.RepeatUnmark;
+import Chat.exceptions.InvalidIndex;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import java.util.ArrayList;
+import java.io.File;
+
+
+public class Evelyn {
+    public static ArrayList<Task> tasks;
+    public static int indexOfTask = 0;
+
+    public static void markTask(int index, boolean done) throws InvalidIndex, RepeatMark, RepeatUnmark {
```

Comment:
Consider making your done variable sound more like a boolean type. maybe name it isDone?